database/migrations/2022_12_27_165111_add_term_start_date_and_term_end_date_to_users_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddTermStartDateAndTermEndDateToUsersTable extends Migration
```

Comment:
The file name and class name should be corrected as we are adding these columns in invoice table not user.
```suggestion
class AddTermStartDateAndTermEndDateToInvoicesTable extends Migration
```
Also correct the file name

Reply:
Also can you add details about second action item in the PR description.
<img width="864" alt="image" src="https://user-images.githubusercontent.com/54376313/209666129-3062a940-ce32-428d-ae90-6848e9d77d69.png">